packages/netlify-cms-core/src/components/Editor/EditorToolbar.js

```diff
@@ -303,16 +303,25 @@ class EditorToolbar extends React.Component {
 
   renderWorkflowStatusControls = () => {
     const { isUpdatingStatus, onChangeStatus, currentStatus, t, useOpenAuthoring } = this.props;
+
+    let buttonText = '';
+
+    if (currentStatus === status.get('DRAFT')) {
+      buttonText = t('editor.editorToolbar.draft');
+    } else if (currentStatus === status.get('PENDING_REVIEW')) {
+      buttonText = t('editor.editorToolbar.inReview');
+    } else if ( currentStatus === status.get('PENDING_PUBLISH')) {
+      buttonText = t('editor.editorToolbar.ready');
+    } else {
+      buttonText = isUpdatingStatus ? t('editor.editorToolbar.updating') : t('editor.editorToolbar.setStatus');
+    }
+
     return (
       <ToolbarDropdown
         dropdownTopOverlap="40px"
         dropdownWidth="120px"
         renderButton={() => (
-          <StatusButton>
-            {isUpdatingStatus
-              ? t('editor.editorToolbar.updating')
-              : t('editor.editorToolbar.setStatus')}
-          </StatusButton>
+          <StatusButton>{`Current Stauts: ${buttonText}`}</StatusButton>
```

Comment:
We should not use hard coded strings. Instead add a translation message and key to:
https://github.com/netlify/netlify-cms/blob/574025fd649d49f70650a6112617642a6cb24f07/packages/netlify-cms-locales/src/en/index.js

and use the `t` to get the translation (see other usages in this file).